django_mysql/models/functions.py

```diff
@@ -260,6 +260,17 @@ def __init__(self, expression, path=None, **extra):
         super(JSONLength, self).__init__(*exprs, output_field=output_field)
 
 
+class JSONValue(Func):
+    function = 'CAST'
+    template = '%(function)s(%(expressions)s as json)'
+    arity = 1
+
+    def __init__(self, expression):
+        import json
```

Comment:
inner import not required, do it at the module level to make the `__init__` faster